tapcfg/config.go

```diff
@@ -180,7 +180,7 @@ var (
 	// required in lnd to run tapd.
 	minimalCompatibleVersion = &verrpc.Version{
 		AppMajor: 0,
-		AppMinor: 15,
+		AppMinor: 17,
```

Comment:
Leave a TODO for the bump to 0.18 once tagged?

Reply:
Yes. And I think we should either remove the very first commit of the PR (or change the versions to current `lnd master`) _OR_ set this to `0.18.0` directly and then update the `go.mod` once the RC1 has been tagged.
Either way we need to park this PR for a bit until v0.18 is at least in RC.